.github/workflows/go.yml

```diff
@@ -1,14 +1,17 @@
 name: Go Build
-on: [push]
+on:
+  push:
+    branches: ["master"]
+  pull_request:
 jobs:
   build:
     name: Build
     runs-on: ubuntu-latest
     steps:
-    - name: Set up Go 1.13
+    - name: Set up Go 1.15
```

Comment:
maybe remove the version at all?

Reply:
I like deterministic builds. Pinning the minor version is the best we can do.

Reply:
I'm commenting L11 only here. just the name

Reply:
I removed it. Sorry answered on my phone. Didn't saw the complete code.